src/ir/effects.h

```diff
@@ -79,23 +88,23 @@ struct EffectAnalyzer : public PostWalker<EffectAnalyzer> {
       return true;
     }
     for (auto local : localsWritten) {
-      if (other.localsWritten.count(local) || other.localsRead.count(local)) {
+      if( other.hasLocalWrite(local) || other.hasLocalRead(local) ){
```

Comment:
whitespace changes are unnecessary (our convention is `if (`, space after the `if`).